wagtail_transfer/locators.py

```diff
@@ -127,6 +127,12 @@ def uid_from_json(self, json_uid):
     def find(self, uid):
         # pair up field names with their respective items in the UID tuple, to form a filter dict
         # that we can use for an ORM lookup
+        if type(uid) == tuple:
+            filters = dict(zip(self.fields, uid))
+        elif type(uid) == str:
+            # if lookup fields are configured for wagtailcore.page, then in the admin view those
+            # fields get passed along as a string
+            filters = dict(zip(self.fields, uid.split(",")))
         filters = dict(zip(self.fields, uid))
```

Comment:
It looks like this line would just override the new filters

Reply:
ah yeah, that may just be a typo from when I copied my tested changes over to the patch :-)